templates/main.template.yaml

```diff
@@ -1696,22 +1577,29 @@ Resources:
             !GetAtt CreateCommonResources.Outputs.BucketName,
             !Ref Section6cS3AuditLogsBucketName,
           ]
-        BucketPrefix: !Ref Section91cS3LoggingBucketPrefix
-        FilterExpression: !Ref Section91dS3LoggingFilterExpression
-        QSS3BucketName: !Ref QSS3BucketName
+        BucketPrefix: !Ref Section10cS3LoggingBucketPrefix
+        FilterExpression: !Ref Section10dS3LoggingFilterExpression
+        QSS3BucketName: !If [call_nfw_stack, !GetAtt SumoFireWallAppsStack.Outputs.QSS3BucketName,
```

Comment:
Why are we doing this ? QSS3BucketName is aws-ia and it will be the default bucket passed as a parameter in main.template ?

Reply:
Q3 BucketName value does not change. it's just replaced by IF to accommodate stack-by-stack creation